java/connector-node/risingwave-connector-service/pom.xml

```diff
@@ -106,5 +106,20 @@
             <artifactId>risingwave-sink-cassandra</artifactId>
             <scope>provided</scope>
         </dependency>
+        <dependency>
+            <groupId>com.risingwave</groupId>
+            <artifactId>risingwave-sink-mock-flink-runtime</artifactId>
```

Comment:
The `mock-runtime` should not be `provided` and it should be packed into the jar of connector service.

The `mock-flink-common` is unnecessary because we don't directly use it. It is included in the `runtime` or `mock-http`